src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+abstract class Figure implements AreaCalculator, Drawable {
+    private static final double MAX_SIZE = 10.0;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private String color;
+    private final Random random = new Random();
```

Comment:
```suggestion
    private String color;
```

Reply:
still not fixed :( I'm waiting for you to visit the next q&a. Let's review your code together